src/utilities.js

```diff
@@ -250,3 +251,92 @@ export function getResults(request) {
     });
   };
 }
+
+export function getTrendingResults(request) {
+  return (searchType, obj) => {
+    const map = {
```

Comment:
Would you mind making this name a little more verbose? `map` is kind of an overloaded term - maybe something like `searchTypeMap` or whatever you think is appropriate